insights/core/dr.py

```diff
@@ -1045,8 +1048,12 @@ def run_components(ordered_components, components, broker):
         except ParseException as pe:
             log.warning(pe)
             broker.add_exception(component, pe, traceback.format_exc())
-        except SkipComponent:
-            pass
+        except SkipComponent as sc:
+            if broker.cli_args and broker.cli_args.show_skips:
+                log.warning(sc)
+                broker.add_exception(component, sc, traceback.format_exc())
+            else:
+                pass
```

Comment:
Is it enough to add it only here? [`insights.core.plugins.parser`](https://github.com/RedHatInsights/insights-core/blob/master/insights/core/plugins.py#L171) duplicates this logic. Would it make sense to add it there as well?

I am not sure if the logic is duplicated elsewhere. I didn't find anything but it doesn't necessarily mean it isn't. :shrug:

Btw. Would you agree that it might be worth finding a way to avoid this code duplication (not in this PR).